model/index.js

```diff
@@ -2,10 +2,12 @@
 'use strict';
 
 const _ = require('lodash');
-const request = require('request');
+const axios = require('axios').default;
 const url = require('url');
 const EventEmitter = require('events').EventEmitter;
 
+const axiosSetting = require('./apis/axios-settings');
```

Comment:
Did you meant to have empty newline previously?  I think the common approach is to put all your variables together

Reply:
I have removed the empty line now